src/libraries/System.Runtime.InteropServices/gen/DllImportGenerator/DllImportGenerator.cs

```diff
@@ -364,8 +364,10 @@ private static TargetFramework DetermineTargetFramework(Compilation compilation,
                 switch (namedArg.Key)
                 {
                     default:
-                        Debug.Fail($"An unknown member '{namedArg.Key}' was found on {attrData.AttributeClass}");
```

Comment:
Should we instead convert to `Debug.WriteLine()`?

Reply:
Yeah, I'm fine with doing that.